docs/api/products-api.md

```diff
@@ -0,0 +1,80 @@
+## 1.GET /api/products/all
+<a name="api-products-get" hidden></a>
+
+| Route                 | Description                        | Access      |
+|:----------------------|:-----------------------------------|:------------|
+| GET /api/products/all | UI Get All Products                | Public      |       
+
+## Params
+1. _id
+    - type: string
+    - required: true
+    - regexp: mongoose_id
+2. name
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 100
+        2. startWith: [a-z] || [A-Z]
+3. type
+    - type: string
+    - required: true
+    - regexp: 
+        1. length: min: 1 - max: 10
+        2. startWith: [a-z] || [A-Z]
+        3. oneOf: ['food','drinks']
+4. image
+    - type: string
+    - required: optional
+    - regexp: length: min: 1 - max: 1000
+5. price
+    - type: double
+    - required: true
+    - regexp: length: min: 3
+6. description 
+    - type: string
+    - required: optional
+    - regexp: length: min: 50 - max: 150
```

Comment:
**Param length**
I think we should delete the minimum length validation due to some conveniences I've just firgured out
min: 50 -> *undefined*

Reply:
Changing to 10 character is better. Because products should have description.